### DISCUSSION TITLE
Could Not Find C:\Kratos\build\RelWithDebInfo\cmake_install.cmake

### DISCUSSION THREAD

**Discussion by engrjessie** [KratosMultiphysics/Kratos#12260] [Q&A]
Hello, I am trying to install Kratos on my end. However I get this error everytime and I do not know what might be wrong. I've tried looking for solution and also trace down the scripts but I find nothing helpful. I would like to ask if you can help me about this. Thank you.

![image](https://github.com/KratosMultiphysics/Kratos/assets/129816555/43b2932a-9123-49a7-99bd-9758110c2d0c)

**Comment by engrjessie** [comment]
The modified configure.bat file is as follows:

@echo off
rem Please do not modify this script

rem For any question please contact with us in:
rem  - https://github.com/KratosMultiphysics/Kratos

rem Optional parameters:
rem You can find a list will all the compiation options in INSTALL.md or here:
rem  - https://github.com/KratosMultiphysics/Kratos/wiki/Compilation-options

rem Set compiler
set CC=cl.exe
set CXX=cl.exe

rem Set variables
if not defined KRATOS_SOURCE set KRATOS_SOURCE=%~dp0..
if not defined KRATOS_BUILD set KRATOS_BUILD=%KRATOS_SOURCE%/build

rem Warning: In windows this option only works if you run through a terminal with admin privileges
set KRATOS_INSTALL_PYTHON_USING_LINKS=ON

rem Set basic configuration
if not defined KRATOS_BUILD_TYPE set KRATOS_BUILD_TYPE=RelWithDebInfo
if not defined BOOST_ROOT set BOOST_ROOT=C:\boost_1_78_0
if not defined PYTHON_EXECUTABLE set PYTHON_EXECUTABLE=C:\Windows\py.exe

rem Set applications to compile
set KRATOS_APP_DIR=applications
set KRATOS_APPLICATIONS=
CALL :add_app %KRATOS_APP_DIR%\LinearSolversApplication;
CALL :add_app %KRATOS_APP_DIR%\StructuralMechanicsApplication;
CALL :add_app %KRATOS_APP_DIR%\FluidDynamicsApplication;
CALL :add_app %KRATOS_APP_DIR%\FemToDemApplication;
CALL :add_app %KRATOS_APP_DIR%\ContactStructuralMechanicsApplication;
CALL :add_app %KRATOS_APP_DIR%\DEMApplication;
CALL :add_app %KRATOS_APP_DIR%\MeshingApplication;
CALL :add_app %KRATOS_APP_DIR%\PfemFluidDynamicsApplication;
CALL :add_app %KRATOS_APP_DIR%\SolidMechanicsApplication;

rem Clean
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\cmake_install.cmake"
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeCache.txt"
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeFiles"

rem Configure
@echo on
cmake -G"Ninja"                                                         ^
-H"%KRATOS_SOURCE%"                                                     ^
-B"%KRATOS_BUILD%/%KRATOS_BUILD_TYPE%"                                  ^
-DCMAKE_BUILD_TYPE=%KRATOS_BUILD_TYPE%                                  ^
-DCMAKE_PDB_OUTPUT_DIRECTORY=%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%/PDB     ^
-DUSE_EIGEN_MKL=OFF                                                     ^
-DKRATOS_GENERATE_PYTHON_STUBS=ON

rem Build
cmake --build "%KRATOS_BUILD%/%KRATOS_BUILD_TYPE%" --target install
goto:eof

rem Function to add apps
:add_app
set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%1;
goto:eof

**Comment by engrjessie** [comment]
Hoping that I could have some answers on what might be the reason why it fails to create the "cmake_install.cmake", so that I can proceed and explore Kratos for my research. Thank you so much in advance. More power!

**Comment by engrjessie** [comment]
I have already resolve the issue.

**Reply by Fe-Marino** [reply]
Hello, what did you do to solve this problem?
I had the same problem.
